guides/v2.3/comp-mgr/install-extensions/inventory-management-installation.md

```diff
@@ -58,6 +58,11 @@ The Inventory Management installation process makes the following changes to the
         'Magento_InventorySourceSelection' => 1,
         'Magento_InventorySourceSelectionApi' => 1,
         'Magento_InventoryShippingAdminUi' => 1,
+        'Magento_InventoryDistanceBasedSourceSelection' => 1,
+        'Magento_InventoryDistanceBasedSourceSelectionAdminUi' => 1,
+        'Magento_InventoryDistanceBasedSourceSelectionApi' => 1,
+        'Magento_InventoryElasticSearch' => 1,
+        'Magento_InventorySetupFixtureGenerator' => 1,
 ```
 
 ## Enable Inventory Management features
```

Comment:
Link not rendering in Disable Inventory Management section

![screen shot 2019-02-13 at 7 38 00 am](https://user-images.githubusercontent.com/14031342/52715575-61770000-2f62-11e9-9ee1-cb1d9384f073.png)

Reply:
It's just a preview error in GH. That's common for page.baseurl having spaces around it. Removed spaces.